java/test/jmri/util/LoggingTestUtil.java

```diff
@@ -0,0 +1,151 @@
+package jmri.util;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.util.*;
+
+import javax.annotation.Nonnull;
+
+import jmri.util.exceptionhandler.UncaughtExceptionHandler;
+
+import org.apache.log4j.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Common utility methods for working with Log4J in tests.
+ * <p>
+ * Two system properties influence how logging is configured in JMRI:
+ * <dl>
+ * <dt>jmri.log</dt><dd>The logging control file. If this file is not an
+ * absolute path, this file is searched for in the following order:<ol>
+ * <li>JMRI settings directory</li>
+ * <li>JMRI installation (program) directory</li>
+ * </ol>
+ * If this property is not specified, the logging control file
+ * <i>default.lcf</i> is used, following the above search order to find it.
+ * </dd>
+ * <dt>jmri.log.path</dt><dd>The directory for storing logs. If not specified,
+ * logs are stored in the JMRI preferences directory.</dd>
+ * </dl>
+ *
+ * @author Bob Jacobsen Copyright 2009, 2010
+ * @author Randall Wood Copyright 2014, 2020
+ */
+public class LoggingTestUtil {
```

Comment:
Why a new class and not adding logging initialization to `jmri.util.JUnitLoggingUtil`? (its likely going to be confusing having `JUnitLoggingUtil` and `LoggingTestUtil`)

Reply:
LoggingTestUtil handles initiation of the logger.

JUnitLoggingUtil is, per the the class description, utilities for handling special case logging messages in tests.

Putting both tasks in a single class violates the single responsibility principle.

I would have named the new class Log4JUtil, but there is a, now deprecated, class with that name already in the package.

Reply:
@rhwood I renamed the class TesttingLoggerConfiguration. That should eliminate the confusion.